enhancements/hypershift/topology-aware-scheduling/topology-aware-scheduling.md

```diff
@@ -0,0 +1,256 @@
+---
+title: topology-aware-scheduling
+authors:
+  - "@Tal-or"
+reviewers: 
+  - "@ffromani"
+  - "@csrwng"
+  - "@sjenning"
+approvers: 
+  - "@jparrill"
+  - "@alvaroaleman"
+  - "@csrwng"
+  - "@sjenning"
+api-approvers:
+  - None
+creation-date: 2024-08-29
+last-updated: 2024-08-29
+tracking-link: 
+  - https://issues.redhat.com/browse/CNF-11154
+see-also:
+  - https://github.com/openshift-kni/numaresources-operator/tree/main
+---
+
+# NumaResources-Operator Deployment on a Hosted Clusters
+
+## Summary
+
+Topology-Aware Scheduling (TAS) enhances Kubernetes/OpenShift (K8S/OCP) scheduler 
+by adding NUMA awareness, enabling more granular scheduling decisions 
+with respect to NUMA resource locality.
```

Comment:
can you include links to Topology-Aware Scheduling docs and the Numa-Resources-Operator?

Reply:
Done. Thanks